testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt

```diff
@@ -27,6 +26,7 @@ import com.earth.testomania.core.developers
 import com.ramcosta.composedestinations.annotation.Destination
 import kiwi.orbit.compose.ui.controls.ChoiceTile
 import kiwi.orbit.compose.ui.controls.Icon
+import kiwi.orbit.compose.ui.controls.Text
```

Comment:
Does this import change cause any visual changes?

Reply:
Nop. I already checked